vue/wolox-books/src/components/InputText.vue

```diff
@@ -0,0 +1,61 @@
+<template lang='pug'>
+  .input-text-container
```

Comment:
Avoid defining components with no logic use common scss classes not vue components